### DISCUSSION TITLE
How to put custom jasmine test timeout

### DISCUSSION THREAD

**Discussion by NavyMint** [webdriverio/webdriverio#7334] [Q&A] [answered]
Hey guys :) I have a question about custom execution timeouts on hooks. Accordingly to jasmine docs we can put second parameter in beforeall and others as timeout on how long hook can be executed. And it worked this way in protractor project. But after migration when Im passing second parameter it treats as retry number. Can you please suggest how can we specify custom timeout for hook/test case and turn off all the retries ?

example I've tried locally:
```
describe('Suite', () => {
    beforeAll(() => {
        console.log('We are in before all');
        throw new Error('Error in before all');
    }, 3);

    it('test', () => {
        console.log('we are in the test case');
    });
});
```

![image](https://user-images.githubusercontent.com/26191692/130790686-a460ce55-92de-4f9a-b1f2-e8e9bf493f82.png)

**Comment by christian-bromann** [comment, marked as answer]
Thanks for reporting, I created an issue for it here: #7339

**Reply by NavyMint** [reply]
thanks a mile, Christian. I will folow issue thread for solution

**Reply by NavyMint** [reply]
I saw fix in 7.11 version, many many thanks
